<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<!-- saved from url=(0066)res_proteomics.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Pandey Lab  </title>

<meta name="description" content="The Pandey Lab at Johns Hopkins University is an interdisciplinary group of molecular biologists, mass spectrometrists and computational biologists who are interested in systems biology.">
<style type="text/css">
<!--
body {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
.hometext {  font-family: "Trebuchet MS"; font-size: 8pt; font-style: normal; font-weight: normal; color: #333333}
.body {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
p {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
-->
</style>
<script language="JavaScript">
<!--
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.0
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && document.getElementById) x=document.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
   
     <!-- Insert additional headers here. -->
   
</head>

<body bgcolor="#FFFFFF" text="#000000" leftmargin="0" topmargin="0" rightmargin="0" bottommargin="0" marginwidth="0" marginheight="0" onload="MM_preloadImages(&#39;images/people_over.gif&#39;,&#39;images/publications_over.gif&#39;,&#39;images/technologies_over.gif&#39;,&#39;images/links_over.gif&#39;,&#39;images/contact_over.gif&#39;,&#39;images/sigtransduction_over.gif&#39;,&#39;images/massspec_over.gif&#39;,&#39;images/bioinformatics_over.gif&#39;,&#39;images/compgenomics_over.gif&#39;)">
<table width="100%" border="0" cellspacing="0" cellpadding="0">
  <tbody><tr> 
    <td> 
      <div align="center"> 
        <table width="100%" border="0" cellpadding="0" cellspacing="0" align="center" bgcolor="#FFFFFF">
          <tbody><tr> 
            <td height="110" valign="top" colspan="3"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="191" height="110" valign="top"> 
                    <div align="left"><img src="images//molecule_topleft.jpg" width="268" height="110"></div>
                  </td>
                  <td width="100%" valign="top">&nbsp;</td>
                  <td width="215" valign="top" align="right"><a href="index.html"><img src="images//logo_pandeylab.gif" width="215" height="110" border="0"></a></td>
                </tr>
              </tbody></table>
            </td>
          </tr>
          <tr>
            <td width="100%" height="108" valign="top" align="left" colspan="3"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="266" height="108" valign="top" align="left"><img src="images//horzbar_imgleft.jpg" width="268" height="108"></td>
                  <td width="512" height="108" valign="top"><div><img src="images//simg_bioinformatics.jpg" width="512" height="108"></div></td>
                  <td width="100%" height="108" valign="top" background="images//horzbar_imgbg.jpg">&nbsp;</td>
                </tr>
              </tbody></table>
            </td>
          </tr>
          <tr> 
            <td valign="top" align="left" width="268"> 
              <table width="268" border="0" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF">
                <tbody><tr> 
                  <td width="268" height="110" valign="top" bgcolor="#FFFFFF"><img src="images//molecule_bottom.jpg" width="268" height="110"></td>
                </tr>
                <tr> 
                  <td valign="top" height="166" bgcolor="#FFFFFF"><a href="res_genomics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/genomics_over.gif&#39;,1)"><img name="sig trans" border="0" src="images//genomics_up.gif"><br>
                    </a><a href="res_proteomics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image19&#39;,&#39;&#39;,&#39;images/proteomics_over.gif&#39;,1)"><img name="Image19" border="0" src="images/proteomics_over.gif"><br>
                    </a><a href="res_bioinformatics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image20&#39;,&#39;&#39;,&#39;images/bioinformatics_over1.gif&#39;,1)"><img name="Image20" border="0" src="images//bioinformatics_up1.gif"><br>
                    </a><a href="res_proteomics.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image19&#39;,&#39;&#39;,&#39;images/massspec_over.gif&#39;,1)"> 
                    </a><a href="res_proteomics.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/sigtransduction_over.gif&#39;,1)"> 
                    </a><!--<a href="res_compgenomics.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_swapImage('Image21','','images/compgenomics_over1.gif',1)"><img name="Image21" border="0" src="images/compgenomics_up1.gif"><br>
                    </a>--><a href="res_proteomics.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/sigtransduction_over.gif&#39;,1)"> 
                    </a> </td>
                </tr>
              </tbody></table>
            </td>
            <td height="35" valign="top" align="left" width="100%"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0" background="images//nav_bg.gif">
                <tbody><tr> 
                  <td width="100%" height="35" valign="top"><a href="people.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;people&#39;,&#39;&#39;,&#39;images/people_over.gif&#39;,1)"><img name="people" border="0" src="images//people_up.gif" width="65" height="35"></a><a href="publications.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;publications&#39;,&#39;&#39;,&#39;images/publications_over.gif&#39;,1)"><img name="publications" border="0" src="images//publications_up.gif" width="124" height="35"></a><a href="technologies.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;technologies&#39;,&#39;&#39;,&#39;images/technologies_over.gif&#39;,1)"><img name="technologies" border="0" src="images//technologies_up.gif" width="130" height="35"></a><a href="links.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;links&#39;,&#39;&#39;,&#39;images/links_over.gif&#39;,1)"><img name="links" border="0" src="images//links_up.gif" width="70" height="35"></a><a href="contact.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;contact&#39;,&#39;&#39;,&#39;images/contact_over.gif&#39;,1)"><img name="contact" border="0" src="images//contact_up.gif" width="123" height="35"></a></td>
                </tr>
              </tbody></table>
                    <div>
<table width="100%" border="0">
                        <tbody><tr>
                          <td style="border-bottom:#387A9A thick inset;" align="center">&nbsp;</td>
                        </tr>
                        <tr>
                          <td style="border-bottom:#387A9A thick inset;  color:#387A9A; font-size:18px; " align="center"><strong style="font:&#39;Times New Roman&#39;, Times, serif;">PROTEOMICS</strong></td>
                        </tr>
                      </tbody></table>
                    <blockquote> 
                      <p><strong><font color="#333366">Mass Spectrometry</font>
                      </strong></p><b> </b><p align="justify">
We are using state-of-the-art mass spectrometric methods for identification of proteins and for characterizing post-translational modifications in signal transduction pathways and in the analysis of several different cancers. We are using different techniques that help us quantitate levels of protein as well as post-translational modifications. We are also developing computational methods that allow us to extract annotated information from existing databases including genomic databases while performing searches using mass spectrometry-derived data.</p>

 <p><strong><font color="#333366">Cancer Biomarkers</font>
                      </strong></p>
 <b> </b><p align="justify">
Our lab is involved in identification  of biomarkers and therapeutic targets of a number of cancers including pancreatic  cancer, hepatocellular carcinoma, breast cancer and cholangiocarcinoma. We use  SILAC and iTRAQ based strategies to identify differentially regulated proteins  between non-neoplastic and neoplastic conditions of these cancers. Candidate  molecules are further validated using tissue microarrays to determine the  prevalence. 
  </p>

                      <ul type="disc">
                        <li><font size="2">Pancreatic Cancer</font>
                          <p align="justify">Pancreatic cancer is one of the  leading causes of cancer deaths in the United States. High mortality rates in  pancreatic cancer have been attributed to late clinical presentation and lack  of effective therapies. We are using high resolution mass spectrometry to carry  out proteomic profiling studies to identify biomarkers and therapeutic targets  of pancreatic cancer. Our efforts include proteomic profiling of pancreatic  juice (<em>Gronborg et al.</em>), secretome  analysis (<em>Gronborg et al.</em>) to  identify biomarkers, phosphoproteomic profiling to identify activated signaling  pathways (<em>Harsha et al.</em>) and  compilation of candidate biomarkers (<em>Harsha  et al.</em>) and molecular alterations reported in the context of pancreatic  cancers (<em>Ranganathan et al.</em>).  </p>
                        </li>
                      </ul>
                      <ul type="disc">
                        <li><font size="2">Hepatocellular Carcinoma</font>
                      <p align="justify">Hepatocellular carcinoma (HCC) is  the fifth most common cancer worldwide. Early diagnosis of HCC in known to  improve the outcome with appropriate clinical intervention. Our lab has  previously published iTRAQ based quantitative proteomic efforts to identify  candidate biomarkers of hepatocellular carcinoma (<em>Chaerkady et al.</em>). Further, we have also identified differentially  regulated glycoproteins in hepatocellular carcinoma using 18O based quantitative  proteomic strategy (<em>Chaerkady et al.</em>).</p>
					  </li>
                      </ul>
                      <ul type="disc">
                        <li><font size="2">Cholangiocarcinoma</font>
                      <p align="justify">Cholangiocarcinoma is the second  most common form of hepatic tumor and accounts for 3% of all gastrointestinal  cancers in the world. Early detection of cholangiocarcinoma offers best hope  for cure. However, the existing markers to diagnose cholangiocarcinoma suffer  from lack of specificity and sensitivity. Towards this end, we carried out  proteomic profiling of bile from cholangiocarcinoma patients to identify  secreted biomarkers (<em>Kristiansen et al.</em>).  We have also carried out 18O based quantitative proteomic study to  identify membrane markers differentially expressed in cholangiocarcinoma as  compared to non-neoplastic biliary tract tissue (<em>Kristiansen et al.)</em>.</p>
                     </li>
                      </ul> 
					   <p><strong><font color="#333366">Stem Cell Proteomics</font>
                      </strong></p>
 <b> </b><p align="justify">
Embryonic stem cell  differentiation to neuronal cells or astrocytes involves multiple stages.  Quantitative proteomics was carried out to map the temporal events during  embryonic stem cell differentiation to neuronal cells, the results of which  were published in <em>Journal of proteome  research</em> (<em>Chaerkady et al.</em>)<em>.</em>&nbsp;  In this study we took advantage of multiplex quantitative proteomic  analysis, capable of analyzing eight different samples simultaneously to  monitor the temporal dynamics of protein abundance as human embryonic stem  cells differentiate into motor neurons or astrocytes .This study was funded by  Maryland stem cell research foundation. </p>
 <p align="justify"> In collaboration with Institute  of Stem cell engineering lab at Hopkins, a study was done to identify the  differences between embryonic stem cells and embryonal carcinoma cells .Though  the two cell types are pluripotent, being derived from teratocarcinomas,  embryonic carcinoma cells differ from embryonic stem cells in their oncogenic  potential. We were able to identify differences that contribute to oncogenicity  versus pluripotency. A summary of identified results were published in the  Journal <em>Proteomics </em>(<em>Chaerkady et al.</em>)</p>
   <p><strong><font color="#333366">miRNA Targets</font>
                      </strong></p>
 <b> </b><p align="justify">
We have used a quantitative proteomic approach to identify  targets of miR-21 at the protein level. miR-21 is a miRNA that has been  reported to be associated with multiple cancer-related  processes including proliferation, apoptosis, invasion, and metastasis. This  study was published in the journal <em>Proteomics (Yi et al) </em>.Although  only 4 targets of this miRNA were known when we initiated our studies, we  identified 58 putative targets from a single proteomics experiment, several of  which we were able to validate using luciferase assays. Given the success of  this approach, we are embarking on carrying out systematic proteomic analyses  to identify targets of miRNAs that are involved in cancers. 
  </p>
  </blockquote>


</div>
            </td>
            <td width="44" height="19" valign="top"><img src="images//white_topmiddle.gif" width="44" height="1"></td>
          </tr>
          <tr> 
            <td height="19" valign="top">&nbsp; 
              
            </td>
            <td height="19" width="100%" valign="top"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="100%" height="19" valign="top" background="images//lowerbar.gif">&nbsp;</td>
                </tr>
              </tbody></table>
            </td>
            <td width="44" height="19" valign="top"><img src="images//white_topmiddle.gif" width="44" height="1"></td>
          </tr>
        </tbody></table>
      </div>
    </td>

  </tr>
</tbody></table>


</body></html>