<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<!-- saved from url=(0064)technologies.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Pandey Lab  Technologies</title>

<meta name="description" content="The Pandey Lab at Johns Hopkins University is an interdisciplinary group of molecular biologists, mass spectrometrists and computational biologists who are interested in systems biology.">
<style type="text/css">
<!--
body {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
.hometext {  font-family: "Trebuchet MS"; font-size: 8pt; font-style: normal; font-weight: normal; color: #333333}
.body {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
p {  font-family: "Trebuchet MS"; font-size: 9pt; font-style: normal; font-weight: normal; color: #333333}
-->
</style>
<script language="JavaScript">
<!--
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.0
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && document.getElementById) x=document.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
   
     <!-- Insert additional headers here. -->
   
</head>

<body bgcolor="#FFFFFF" text="#000000" leftmargin="0" topmargin="0" rightmargin="0" bottommargin="0" marginwidth="0" marginheight="0" onload="MM_preloadImages(&#39;images/people_over.gif&#39;,&#39;images/publications_over.gif&#39;,&#39;images/technologies_over.gif&#39;,&#39;images/links_over.gif&#39;,&#39;images/contact_over.gif&#39;,&#39;images/sigtransduction_over.gif&#39;,&#39;images/massspec_over.gif&#39;,&#39;images/bioinformatics_over.gif&#39;,&#39;images/compgenomics_over.gif&#39;)">
<table width="100%" border="0" cellspacing="0" cellpadding="0">
  <tbody><tr> 
    <td> 
      <div align="center"> 
        <table width="100%" border="0" cellpadding="0" cellspacing="0" align="center" bgcolor="#FFFFFF">
          <tbody><tr> 
            <td height="110" valign="top" colspan="3"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="191" height="110" valign="top"> 
                    <div align="left"><img src="images/molecule_topleft.jpg" width="268" height="110"></div>
                  </td>
                  <td width="100%" valign="top">&nbsp;</td>
                  <td width="215" valign="top" align="right"><a href="index.html"><img src="images/logo_pandeylab.gif" width="215" height="110" border="0"></a></td>
                </tr>
              </tbody></table>
            </td>
          </tr>
          <tr>
            <td width="100%" height="108" valign="top" align="left" colspan="3"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="266" height="108" valign="top" align="left"><img src="images/horzbar_imgleft.jpg" width="268" height="108"></td>
                  <td width="512" height="108" valign="top"><div><img src="images/horzbar_imgright.jpg" width="512" height="108"></div></td>
                  <td width="100%" height="108" valign="top" background="images/horzbar_imgbg.jpg">&nbsp;</td>
                </tr>
              </tbody></table>
            </td>
          </tr>
          <tr> 
            <td valign="top" align="left" width="268"> 
              <table width="268" border="0" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF">
                <tbody><tr> 
                  <td width="268" height="110" valign="top" bgcolor="#FFFFFF"><img src="images/molecule_bottom.jpg" width="268" height="110"></td>
                </tr>
                <tr> 
                  <td valign="top" height="166" bgcolor="#FFFFFF"><a href="res_genomics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/genomics_over.gif&#39;,1)"><img name="sig trans" border="0" src="images/genomics_up.gif"><br>
                    </a><a href="res_proteomics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image19&#39;,&#39;&#39;,&#39;images/proteomics_over.gif&#39;,1)"><img name="Image19" border="0" src="images/proteomics_up.gif"><br>
                    </a><a href="res_bioinformatics.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image20&#39;,&#39;&#39;,&#39;images/bioinformatics_over1.gif&#39;,1)"><img name="Image20" border="0" src="images/bioinformatics_up1.gif"><br>
                    </a><a href="technologies.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;Image19&#39;,&#39;&#39;,&#39;images/massspec_over.gif&#39;,1)"> 
                    </a><a href="technologies.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/sigtransduction_over.gif&#39;,1)"> 
                    </a><!--<a href="res_compgenomics.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_swapImage('Image21','','images/compgenomics_over1.gif',1)"><img name="Image21" border="0" src="images/compgenomics_up1.gif"><br>
                    </a>--><a href="technologies.html#" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;sig trans&#39;,&#39;&#39;,&#39;images/sigtransduction_over.gif&#39;,1)"> 
                    </a> </td>
                </tr>
              </tbody></table>
            </td>
            <td height="35" valign="top" align="left" width="100%"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0" background="images/nav_bg.gif">
                <tbody><tr> 
                  <td width="100%" height="35" valign="top"><a href="people.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;people&#39;,&#39;&#39;,&#39;images/people_over.gif&#39;,1)"><img name="people" border="0" src="images/people_up.gif" width="65" height="35"></a><a href="publications.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;publications&#39;,&#39;&#39;,&#39;images/publications_over.gif&#39;,1)"><img name="publications" border="0" src="images/publications_up.gif" width="124" height="35"></a><a href="technologies.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;technologies&#39;,&#39;&#39;,&#39;images/technologies_over.gif&#39;,1)"><img name="technologies" border="0" src="images/technologies_up.gif" width="130" height="35"></a><a href="links.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;links&#39;,&#39;&#39;,&#39;images/links_over.gif&#39;,1)"><img name="links" border="0" src="images/links_up.gif" width="70" height="35"></a><a href="contact.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage(&#39;contact&#39;,&#39;&#39;,&#39;images/contact_over.gif&#39;,1)"><img name="contact" border="0" src="images/contact_up.gif" width="123" height="35"></a></td>
                </tr>
              </tbody></table>
                    <div>
<!--<table width="100%" border="0">
                        <tr>
                          <td style="border-bottom:#387A9A thick double; font:Geneva, Arial, Helvetica, sans-serif; color:#387A9A; font-size:13px" ><strong>TECHNOLOGIES</strong></td>
                        </tr>
                      </table>-->
					 <table width="100%" border="0">
                        <tbody><tr>
                          <td style="border-bottom:#387A9A thick inset;" align="center">&nbsp;</td>
                        </tr>
                        <tr>
                          <td style="border-bottom:#387A9A thick inset;  color:#387A9A; font-size:18px; " align="center"><strong style="font:&#39;Times New Roman&#39;, Times, serif;">TECHNOLOGIES</strong></td>
                        </tr>
                      </tbody></table>
              <table width="100%" cellpadding="10" cellspacing="5" border="0">
                <tbody><tr>
                  <td>
                      <p><br>
                        <font color="#333366"><b>Mass Spectrometry</b></font></p>
                      <p align="justify">The Pandey lab is involved in developing proteomic methods as well in applying mass spectrometry for studying signaling pathways, identifying biomarkers and characterizing protein-protein interactions. The lab is equipped with a number of mass spectrometers including Agilent's 6538 Ultra High Definition Accurate-mass Q-TOF, Thermo's LTQ-Orbitrap XL ETD mass spectrometer and Agilent's 6430 Triple Quadrupole mass spectrometer.</p> 
                  </td>
                </tr>
				<tr>
                  <td style="border:1px #cccccc solid">
                     <p align="justify"><img src="images/Orbitrap_Elite_ETD.jpg" align="left" style="padding: 2 2 2 2; border: solid 1px #9B989A;"><b> 
                        Thermo LTQ Orbitrap Elite ETD mass spectrometer</b></p>
                     <!--p align="justify">The LTQ Orbitrap Elite is one of the Thermo hybrid Fourier Transform mass spectrometers that combines a linear ion trap  mass analyzer with an Orbitrap mass analyzer. In addition to combination of fast scanning linear ion trap and high resolving power and mass accuracy of Orbitrap mass analyzer, this instrument has also equipped with ETD at the backend which allows the orthogonal, electron-based peptide/protein fragmentation, resulting in c/z fragment ions. The LTQ Orbitrap Elite ETD mass spectrometer has the highest resolving power (upto 480,000 of resolution at 400 m/z) with the proteomics grade scan speed. In addition to the conventional CID, the instrument also features an new, additional collision cell for higher energy collisional dissociation (HCD) which is much faster, as compared to any previous versions of HCD cells (e.g. XL and Velos). Maximizing almost all capabilities of previous Orbitrap-based mass spectrometers while providing higher sensitivity, resolution and mass accuracy makes it suitable for a variety of proteomics studies including PTMs analysis. We use the LTQ Orbitrap Elite mass spectrometer for mapping human proteome as a metaproteomics analysis.</p-->
<p align="justify">
We have the LTQ Orbitrap Elite mass spectrometer equipped with ETD on a short term lease in collaboration with Dr. Ted Dawson's laboratory at Johns Hopkins. It is one of Thermo's hybrid Fourier Transform mass spectrometers that combines a linear ion trap mass analyzer with an Orbitrap mass analyzer. In addition to combination of fast scanning linear ion trap and high resolving power and mass accuracy of Orbitrap mass analyzer, this instrument is also equipped with ETD which allows the orthogonal, electron-based peptide/protein fragmentation, resulting in c/z fragment ions. In addition to the conventional CID, the instrument also features a new higher energy collisional dissociation (HCD) cell. We are using this mass spectrometry to study Parkinson's disease.

</p>               

   </td>
                </tr>
				<tr>
                  <td style="border:1px #cccccc solid">
                     <p align="justify"><img src="images/qtof_new.jpg" align="left" style="padding: 2 2 2 2; border: solid 1px #9B989A;"><b> 
                        Agilent's 6538 Ultra High Definition Accurate-mass Q-TOF</b></p>
                     <p align="justify">This mass spectrometer provides a very
                       high resolution (40,000), high mass accuracy (parts per
                       billion) and dynamic
                       range (four to five orders of magnitude). The sensitivity
of detection for peptides is in attomole range. The variable scan speed mode
                       offers
dynamic scan speed based on abundance of precursor peptides. Agilent's 6538 Q-
TOF is interfaced with Agilent's HPLC chip-cube nanospray source which offers
great advantage by reducing variation caused by manual intervention. In our laboratory,
we are
using this mass spectrometer for various proteomic applications such as proteogenomics,
quantitative proteomics based on SILAC, iTRAQ,  <sup>18</sup>O labeling and label-free
approaches and for the analysis of post-translational modification such as acetylation
and
phosphorylation.</p>
                  </td>
                </tr>
                <!--tr>
                  <td style="border:1px #cccccc solid">
                      <p><img src="images/Orbitrap_small.gif" style="padding: 2 2 2 2; border: solid 1px #9B989A;" align="left"><b>Thermo LTQ Orbitrap XL mass spectrometer</b></p><p align="justify">Thermo LTQ Orbitrap XL mass spectrometer is a hybrid Fourier Transform mass spectrometer which combines features of linear ion trap and orbitrap mass analyzer. This combination of fast scanning linear ion trap and high resolving power and mass accuracy of orbitrap mass analyzer makes it one of the best mass spectrometers for proteomic research. The instrument also features an additional collision cell for higher energy collisional dissociation (HCD) which is particularly useful for analyzing ions in the low-mass region of the spectrum. Culminating almost all the capabilities of previous mass spectrometers while providing higher sensitivity, resolution and mass accuracy makes it suitable for a variety of proteomics studies. We use LTQ Orbitrap for both qualitative and quantitative proteomics studies including identification of biomarkers of GI cancers and characterization of post-translational modifications like phosphorylation and glycosylation.</p>
                  </td>
                </tr-->
               <!-- <tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/iontrap_small.gif" width="400" height="300" align="left"><b> 
                        ETD equipped Agilent 6340 ion trap mass spectrometer</b></p><p align="justify">The Agilent 6340 ion-trap LC/MS system provides flexibility to choose between collision induced dissociation (CID) and electron transfer dissociation (ETD) methods. Although CID has been traditionally used as a preferred dissociation method for most proteomic studies involving protein identification, its utility has been limited while analyzing labile post-translational modifications. However, recently introduced electron transfer dissociation (ETD) method is particularly suitable for studying labile post-translational modifications such as phosphorylation and glycosylation of serine and threonine residues. By carrying out alternating fragmentation of a peptide by both CID and ETD, it is now possible to get better sequence coverage of a peptide resulting in confident protein identification. Our lab uses this instrument to study phosphorylation and O-GlcNAcylation of serine/threonine residues and utility of alternating CID/ETD fragmentation for the analysis of nonmodified peptides.</p>
                  </td>
                </tr>
                <tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/QSTAR_small.gif" width="400" height="300" align="left"><strong>Applied Biosystems QSTAR mass spectrometer </strong>                       </p>
                  </td>
                </tr>
                <tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/QTOF_small.gif" align="left"><strong>Micromass
                         Q-TOF Ultima API-US</strong> (quadrupole time of flight mass spectrometer) 
                        equipped with an Agilent Capillary LC pump and a cooled 
                        Micro-well plate autosampler 
                        </p>
                  </td>
                </tr>-->

               				<tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/qqq_new.jpg" style="padding: 2 2 2 2; border: solid 1px #9B989A;" align="left"><b> 
                        Agilent's 6430 Triple Quadrupole LC/MS</b></p>
                     <p align="justify">This mass spectrometer is designed for true multiple reaction monitoring (MRM) based
quantitation. The improved hexapole collision cell offers fast monitoring of multiple
peptides with short dwell time enabling one to monitor hundreds to thousands of
transitions in a single run. We are using this mass spectrometer to develop biomarker
assays in serum, urine and tissues for developing MRM assays for early detection of
cancers and other diseases.</p>
                  </td>
                </tr>
				<tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/offgel_new.jpg" style="padding: 2 2 2 2; border: solid 1px #9B989A;" align="left"><b> 
                        3100 OFFGEL Fractionator </b></p>
                    <p align="justify">Agilent's 3100 OFFGEL fractionator can be used for in-solution pI based separation. Our lab utilizes this fractionation technique for fractionating proteins as well as peptides to obtain a deeper coverage of the proteome in a number of quantitative proteomics and proteogenomics projects. </p>
                  </td>
                </tr>

				
				<tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/lc_new.jpg" style="padding: 2 2 2 2; border: solid 1px #9B989A;" align="left"><b> 
                        HPLC Systems</b></p>
                     <p align="justify">Our lab has one 1200 series and three 1100 series HPLC systems from Agilent Technologies. We use these LC systems for both nano-flow and standard flow applications. In most projects, we carry out offline fractionation of proteins/peptides using various LC based techniques such as strong cation exchange chromatography (SCX), anion exchange chromatography and macroporous reversed phase chromatography. We also employ multiple affinity removal system for depletion of abundant proteins from serum, urine or other body fluids.</p>
                  </td>
                </tr>
				 <!--<tr>
                  <td style="border:1px #cccccc solid">
                     <p><img src="images/Fraction_collector_small.gif" align="left"><strong>Fraction collector</strong></p>
                  </td>
                </tr>-->

        <!--        <tr>
                  <td>
                    <p><font color="#333366"><b>Liquid Chromatography</b></font></p>
                      <p><a href="http://www.chem.agilent.com/Scripts/PDS.asp?lPage=277" target="_blank">Two 
                        CapLC pumps from Agilent and a Micro wellplate auto sampler</a>. 
                        The CapLCs are both connected to the mass spectrometers. 
                        One set-up is a &quot;high throughput&quot; setup consisting 
                        of a CapLC pump and an auto sampler. The other CapLC 
                        pump is used together with off-line &quot;bomb&quot; 
                        loading. <br>
                        All separations are performed using homemade fused silica type columns.</p>
                  </td>
                </tr>
                <tr>
                  <td>
                     <p><font color="#333366"><b>Data Databases</b></font></p>
                      <p>Fragmented peptides are searched using a Mascot cluster 
                        from <a href="http://www.matrixscience.com">Matrixscience</a>.<br>
                        <img src="images/data_databases.gif" width="425" height="240"><br>
                        <font size="1">Schematic of high load flow/low elute flow 
                        set-up. Sample is loaded on to the pre column @5-10&micro;L/min 
                        with the 6-port valve in &quot;waste&quot; position. When 
                        starting to elute, the valve is shifted to &quot;closed&quot; 
                        and simultanusely the flow from the pump is lowered to 
                        approx. 300nL/min. With the 6-port-valve in &quot;closed&quot; 
                        position the flow will forced through the emitter. </font> 
                      </p>
                  </td>
                </tr>-->
              </tbody></table>


</div>
            </td>
            <td width="44" height="19" valign="top"><img src="images/white_topmiddle.gif" width="44" height="1"></td>
          </tr>
          <tr> 
            <td height="19" valign="top">&nbsp; 
              
            </td>
            <td height="19" width="100%" valign="top"> 
              <table width="100%" border="0" cellpadding="0" cellspacing="0">
                <tbody><tr> 
                  <td width="100%" height="19" valign="top" background="images/lowerbar.gif">&nbsp;</td>
                </tr>
              </tbody></table>
            </td>
            <td width="44" height="19" valign="top"><img src="images/white_topmiddle.gif" width="44" height="1"></td>
          </tr>
        </tbody></table>
      </div>
    </td>

  </tr>
</tbody></table>


</body></html>
